soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -251,15 +251,31 @@ void DrawGroupWithBorder(T&& drawFunc) {
     ImGui::EndGroup();
 }
 
+char z2ASCII(int code) {
+    int ret;
+    if (code < 10) { //Digits
```

Comment:
```suggestion
    int ret;
    
    if (code < 10) { //Digits
```
Also, why not make this a `char`?